admin_manual/configuration/server/logging_configuration.rst

```diff
@@ -35,10 +35,54 @@ date format in the example below, the date/time format will be written in the fo
 
 ::
 
-    "log_type" => "owncloud",
-    "logfile" => "owncloud.log",
-    "loglevel" => "3",
-    "logdateformat" => "F d, Y H:i:s",
+    'log_type' => 'owncloud',
+    'logfile' => 'owncloud.log',
+    'loglevel' => 2,
+    'logdateformat' => 'F d, Y H:i:s',
+
+**Log rotation:**
+
+To rotate this log file, use the following key:
+
+::
+
+	'log_rotate_size' => false,
+
+| The default is 0 or false which disables log rotation. 
+| Specify a size in bytes, for example 104857600 
+| (100 megabytes = 100 * 1024 * 1024 bytes). 
+| A new logfile is created with a new name when the old logfile reaches your limit.
+| If a rotated log file is already present, it will be overwritten.
+| If enabled, only the active log file and one rotation file are present.
+
+The file name of the rotated logfile is defined by the key ``logfile`` and a number as extension.
+Example:
+
+:: 
+
+    owncloud.1
+
+In case you want to implement a more sophisticated log rotation, you can use the log rotation 
+machanism of your UINX operating system, see the following example. Please adopt or customize 
```

Comment:
s/UINX/UNIX/g. I'd suggest we don't reference UNIX, but Linux as ownCloud doesn't support UNIX. The closest we would come to supporting UNIX is one of the BSDs; and these aren't officially supported either.